spec/models/item_orders_spec.rb

```diff
@@ -14,7 +14,7 @@
   end
 
   describe 'instance methods' do
-    it 'subtotal' do
+    it '#subtotal' do
```

Comment:
I went through and added the pound sign to a bunch of instance methods in tests that were present when we forked the repo. It doesn't effect the functionality at all, just looks cool and follows the rules.

Reply:
I love rules.